pkgs/by-name/nt/ntfs2btrfs/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, cmake, fmt, lzo, pkg-config, zlib, zstd }:
+
+stdenv.mkDerivation rec {
+  pname = "ntfs2btrfs";
+  version = "20240115";
+
+  nativeBuildInputs = [ cmake fmt lzo pkg-config zlib zstd ];
```

Comment:
The standard ordering puts nativeBuildInputs after src. Also libraries like zlib, fmt, lzo and zstd go to buildInputs.

Reply:
ah got it, i will clean this up!